sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/icd/IcdHeader.java

```diff
@@ -3,91 +3,6 @@
 // SPDX-License-Identifier: Apache-2.0
 package org.lfenergy.compas.sct.commons.scl.icd;
 
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import org.lfenergy.compas.scl2007b4.model.TCompasICDHeader;
-import org.lfenergy.compas.scl2007b4.model.TCompasIEDRedundancy;
-import org.lfenergy.compas.scl2007b4.model.TCompasIEDType;
-import org.lfenergy.compas.sct.commons.exception.ScdException;
+public record IcdHeader(String headerId, String headerVersion, String headerRevision) {
```

Comment:
If the new IcdHeader record is used only by CompasIcdHeader, is it really useful to create that record ?